### ISSUE TITLE
500 error message 

### ISSUE THREAD

**Issue by ericquayson** [CropSwap/crop_swap_old#1]
![screen shot 2558-11-03 at 3 41 41 pm](https://cloud.githubusercontent.com/assets/12155476/10921053/6e40b9e2-8241-11e5-987a-f4dd86c8fb96.png)
- trying to post to the database and I'm getting this error 
- Can you write out documentation for the requests? Also, documentation for the error messages as well?

**Comment by Ru-T** [comment]
YES. Where should I drop it so that it's available to you? Should we have a
shared folder somewhere?

**Comment by ericquayson** [comment]
You can create a wiki and store it in there.

**Comment by Ru-T** [comment]
Ok. Where are you? I'll come join.
